### DISCUSSION TITLE
Standings - Not Displaying

### DISCUSSION THREAD

**Discussion by dawghouse73** [nmelhado/league-page#218] [Q&A]
I get the Hut Hut page not found when I try and access the Standings Page. 
I appreciate we may not have any standings (this is our first season upcoming as we're a Dynasty Start up). 

But I noticed on the Legends site that it states "Preseason, No Standings Yet"
![image](https://github.com/nmelhado/league-page/assets/133917715/cb558d36-fa1e-4f75-b035-e4bc0439f7f8)

Is there a special configuration/update I must do in order to get the standings page working?

Thanks in advance. 
Shulaces Out Dan!

**Comment by nmelhado** [comment]
No this is likely an edge case that only happens when there are no records yet. Needs to be fixed. Could you report an official bug so I can handle this. Definitely include your league ID, so I can reproduce and fix it. Thanks for the report!
